### ISSUE TITLE
Easy selection of content

### ISSUE THREAD

**Issue by rhymdsadeelf** [cbucher/console#312]
Just a suggestion (not sure if it is a good idea). It will be like:

Shift+Arrow will start selection and pressing any other key beside these will automatically copy content and clear selection.

**Comment by cbucher** [comment]
When you write Shift+Arrow, you mean this:
![capture](https://cloud.githubusercontent.com/assets/363759/13197856/3bb01bf8-d7fa-11e5-8c85-e25e2835147b.PNG)
?

**Comment by rhymdsadeelf** [comment]
Yes! That does the first part. What I suggest is an option to press any key beside these to clear selection.

**Comment by cbucher** [comment]
>Shift+Arrow will start selection and pressing any other key beside these will automatically copy content and clear selection.

I find this behavior unusual.
Usually you use a specific hotkey for copying and pressing any other key cancels the selection.

**Comment by rhymdsadeelf** [comment]
Yes! I am completely fine with this. I think it will be much better if selection shortcut can work like how they work in `cmd.exe`
